HW0/P6/P6.txt

```diff
@@ -1 +1,121 @@
 # Your answers here
```

Comment:
Your graph is pretty noisy below 10^-3 seconds, and it is odd to see more than 4x speed-up with only 4 processes.